### ISSUE TITLE
confirm that we need the Google Plus scopes for sign in

### ISSUE THREAD

**Issue by kueda** [inaturalist/INaturalistIOS#114]
For Google sign in we request a number of scopes, including `kGTLAuthScopePlusLogin` and `kGTLAuthScopePlusMe, which generate a kind of scary "iNat wants to access your g+ contacts" message. Please check to see if we really need those, and remove if necessary. I *believe* I added those to get access to the user's email address, and possibly their Picasa account, but those scopes are pretty confusing so I may have overreached.

Low priority.

**Comment by alexshepard** [comment]
https://developers.google.com/+/domains/authentication/scopes

**Comment by alexshepard** [comment]
https://developers.google.com/identity/sign-in/ios/

**Comment by kueda** [comment]
Is that a "yes we need those scopes"?

**Comment by alexshepard** [comment]
No, sorry, that's a note to self about google's new auth library.

**Comment by alexshepard** [comment]
Just trying to make myself notes about what I'll need to dig into to figure this out.

**Comment by alexshepard** [comment]
I need to go in and find out if we still ask for these scopes and if so, if we still need them.
